ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly.yaml

```diff
@@ -4372,6 +4372,62 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-mce-power-guest
+- as: aws-ipi-ovn-hypershift-mce-guest-f7
```

Comment:
4.18 has reached z-stream stage, consider reducing the job frequency, `f9` or `f14`

Reply:
@LiangquanLi930 Any idea ? update to f14 ?